recipes/calcephpy/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "calcephpy" %}
+{% set version = "3.0.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "e8b2e08306412355672ee945173bad3b67ce607558213945e061a86ce2af3107" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: 'python setup.py install '
```

Comment:
Why the quotes? And can you use pip? ` python -m pip install --no-deps --ignore-installed .`

Reply:
I use the command "conda skeleton pypi calceph" to generate the meta.yaml. 
So conda introduces these quotes in the generated script command.

Reply:
Any news for the merge request ?